src/builtin.cc

```diff
@@ -113,6 +113,7 @@ Builtins::Builtins()
 	this->assignments.emplace_back(new Assignment("$vpr", make_shared<Literal>(VectorType(0.0, 0.0, 0.0))) );
 	this->assignments.emplace_back(new Assignment("$vpd", make_shared<Literal>(500)) );
 	this->assignments.emplace_back(new Assignment("$vpf", make_shared<Literal>(22.5)) );
+	this->assignments.emplace_back(new Assignment("$anaglyph", make_shared<Literal>(Value::undefined.clone())) ); // true if rendering anaglyph 3d
```

Comment:
What's the use case for exposing this to the script?

Reply:
We can drop this variable. I have used it like this:
```
color($anaglyph?"Yellow":"Red")
cube(10);
```
but now the dubois algorithm modifies colors automatically, so this variable is no longer needed.